From 3a294b12e9bf9345d48333cad14f752bafafe1a2 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Tue, 9 Jul 2019 17:52:44 +0800
Subject: [PATCH 908/909] arch: arm64: dts: reduce the size of reserved memory

On renesas rcar platform, reserved 3 blocks memory. The total
size is 704MB, they take up most of low address money. This
will cause kdump feature is not able to work. Because there
is no remaining memory for crash kernel.
So far, there is not any feature need to reserve such big memory.
Therefore, reduce the size of the 2 blocks so that there is enough
memory for the crash kernel and make sure the crash kernel
boots up smoothly.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index 87132c0fe4b4..c1a030dca4c1 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -43,14 +43,16 @@
 		/* device specific region for Lossy Decompression */
 		lossy_decompress: linux,lossy_decompress@54000000 {
 			no-map;
-			reg = <0x00000000 0x54000000 0x0 0x03000000>;
+			alloc-ranges = <0x0 0x6d000000 0x0 0x03000000>;
+			size = <0x0 0x03000000>;
 		};
 
 		/* global autoconfigured region for contiguous allocations */
 		linux,cma@57000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x57000000 0x0 0x19000000>;
+			alloc-ranges = <0x0 0x70000000 0x0 0x8000000>;
+			size = <0x0 0x8000000>;
 			linux,cma-default;
 		};
 
@@ -58,7 +60,8 @@
 		mmp_reserved: linux,multimedia@70000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x70000000 0x0 0x10000000>;
+			alloc-ranges = <0x0 0x78000000 0x0 0x8000000>;
+			size = <0x0 0x8000000>;
 		};
 	};
 
-- 
2.17.1

